Determine password length from env variable, if set

dotcs opened the pull request:
With this change, the variable `GOPASS_PW_DEFAULT_LENGTH` can be used to set a default length for the `generate` cmd.

This PR fixes #2190.

dominikschulz commented on internal/action/generate_test.go:
```diff
@@ -178,6 +178,35 @@ func TestGenerate(t *testing.T) { //nolint:paralleltest
+	t.Run("generate --force foobar", func(t *testing.T) { //nolint:paralleltest
+		t.Setenv("GOPASS_PW_DEFAULT_LENGTH", "42")
+
+		if testing.Short() {
```
I don't think you need to skip this test in short mode.

dotcs replied:
Thanks for the hint. I've removed the if-condition. 
I'm not exactly sure when to skip tests in short mode and when not. I was just copying the lines from another test. Sorry, should have mentioned that. Is it okay to leave it for the other test in line 193-204, or should I remove it there as well.

dominikschulz reviewed (dismissed):
LGTM. Thanks a lot!

dotcs commented:
Thanks @dominikschulz for the review! :+1:
I have incorporated some feedback that I received in the linked issue, fixed a few lint bugs and added some more tests. I guess you need to take another look. Thanks in advance.

AnomalRoil reviewed (approved):
Awesome, that looks good.

dotcs commented:
Thanks for the review, @AnomalRoil. I've fixed the remaining lint issue in my latest commit. There are a couple of other lint issues which have nothing to do with my changes in this PR. `make codequality` also fails for me on the `master` branch. Is there an easy way to locally test this and only get those lint errors that are because of the code changes in the PR? And is there any chance that existing lint errors will be fixed in `master` to have a clean result? That would imo make writing PRs much simpler. I hope that this time everything is working out. 🤞

dominikschulz reviewed (approved):
LGTM. Thanks a lot!
